README.md

```diff
@@ -48,46 +45,50 @@ TODO
 
 ## Running the Example
 
-You can use the provided [AWS SAM template] 
+You can use the provided [AWS SAM template](./template.yml) to launch a stack that demonstrates the Lambda file processing reference architecture. Details about the resources created by this template are provided in the *SAM Template Resources* section of this document.
 
-TODO **Insert CFN Template Link** 
 
-to launch a stack that demonstrates the Lambda file processing reference architecture. Details about the resources created by this template are provided in the *CloudFormation Template Resources* section of this document.
+### Using SAM to Build and Deploy the Application
 
-**Important** Because the AWS CloudFormation stack name is used in the name of the Amazon Simple Storage Service (Amazon S3) buckets, that stack name must only contain lowercase letters. Use lowercase letters when typing the stack name. The provided CloudFormation template retrieves its Lambda code from a bucket in the us-east-1 region. To launch this sample in another region, please modify the template and upload the Lambda code to a bucket in that region.
+#### Build
 
+The AWS SAM CLI comes with abstractions for a number of Lambda runtimes to build your dependencies, and copies the source code into staging folders so that everything is ready to be packaged and deployed. The *sam build* command builds any dependencies that your application has, and copies your application source code to folders under aws-sam/build to be zipped and uploaded to Lambda. 
 
-Choose **Launch Stack** to launch the template in the us-east-1 region in your account:
-
-TODO Update Link
-
-[![Launch Lambda File Processing into North Virginia with CloudFormation](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/images/cloudformation-launch-stack-button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=lambda-file-processing&templateURL=https://s3.amazonaws.com/awslambda-reference-architectures/file-processing/lambda_file_processing.template)
+```bash
+sam build --use-container
+```
 
-## Using SAM
+#### Package
 
-Install the dependencies for lambda
+Next, run *sam package*.  This command takes your Lambda handler source code and any third-party dependencies, zips everything, and uploads the zip file to your Amazon S3 bucket. That bucket and file location are then noted in the packaged-template.yaml file. You use the generated packaged-template.yaml file to deploy the application in the next step. 
 
 ```bash
-cd src/conversion && pip install -r requirements.txt -t .
-cd src/sentiment && pip install -r requirements.txt -t .
+sam package \
+    --output-template-file packaged-template.yml \
+    --s3-bucket bucketname 
 ```
 
-Run SAM package (equivalent to aws cloudformation package)
+**Note**
```

Comment:
Another option here is to just create a bucket, if we don't name the resource it would just take the stack name with a random uuid. We can then just output it.